src/common/userExperience.ts

```diff
@@ -187,3 +191,37 @@ export const experienceTypeToRepositoryMap: Record<UserExperienceType, string> =
     [UserExperienceType.Publication]: 'UserPublicationExperience',
     [UserExperienceType.Course]: 'UserCourseExperience',
   };
+
+// Work Email Verification
+export const completeVerificationForExperienceByUserCompany = async (
+  con: DataSource,
+  { companyId, userId, email: verificationEmail }: UserCompany,
+): Promise<boolean> => {
+  if (!companyId) return false;
+
+  const experience = await queryReadReplica(con, ({ queryRunner }) =>
+    queryRunner.manager
+      .getRepository<UserWorkExperience>('UserWorkExperience')
+      .findOneBy({
+        userId,
+        companyId,
+      }),
```

Comment:
you can just do update, no need to check if row exists

Reply:
Updated ✔️